### DISCUSSION TITLE
The error is that the command is duplicated, although I do not have such a command.

### DISCUSSION THREAD

**Discussion by ArtemLakeev** [discordjs/discord.js#9903] [Q&A]
I'm trying to register a command with a slash. I am creating a unique team on my entire server and in the bot. And when you run the code (link: https://guide.discordjs.dev/guide/creating-your-bot/registering-slash-commands ) gives an error that the command is duplicated (Started refreshing application (/) commands.
DiscordAPIError[50035]: Invalid Form Body
1[APPLICATION_COMMANDS_DUPLICATE_NAME]: Application command names must be unique) please help

**Comment by sdanialraza** [comment]
Log the commands you're registering and see what it shows and I would recommend joining the [Discord server](https://discord.gg/djs) as you can get help there much quicker.

**Comment by xero-lib** [comment]
This typically means you have already registered the command with Discord. To fix this, try pushing an empty list to the application commands endpoint:

```js
import { Routes } from 'discord.js';
import { REST } from '@discordjs/rest';

const rest = new REST({ version: '10' }).setToken("<your token here>");

await rest.put(Routes.applicationCommands("<your client ID here>"), { body: [] })
     .then(() => console.log('Successfully deleted all application commands.'))
     .catch(console.error);
```
